client/src/components/Jokes.js

```diff
@@ -0,0 +1,40 @@
+import React from "react";
+import axios from "axios";
+import requiresAuth from "../auth/requiresAuth";
+
+class Jokes extends React.Component {
+  state = {
+    jokes: []
+  };
+
+  componentDidMount() {
+    axios
+      .get("/jokes")
+      .then(res => {
+        this.setState({ jokes: res.data });
+      })
+      .catch(e => {
+        console.error();
+      });
```

Comment:
I assume the error `e` is meant to passed to the inner function here, but why use the extra step? Just use the inner function as your callback:
```javascript
  .catch(console.error);
```

Reply:
simpler